book/src/user/target-tier-policies.md

```diff
@@ -0,0 +1,288 @@
+  tests for the target. If at all possible, this documentation should show how
+  to run Zebra programs and tests for the target using emulation, to allow
+  anyone to do so. If the target cannot be feasibly emulated, the documentation
+  should explain how to obtain and work with physical hardware, cloud systems,
+  or equivalent.
+- The target must document its baseline expectations for the features or
+  versions of CPUs, operating systems, libraries, runtime environments, and
+  similar.
+- Tier 2 targets must not leave any significant portions of `zebra` unimplemented
+  or stubbed out, unless they cannot possibly be
+  supported on the target.
+  - The right approach to handling a missing feature from a target may depend
+    on whether the target seems likely to develop the feature in the future. In
+    some cases, a target may be co-developed along with Zebra support, and Zebra
+    may gain new features on the target as that target gains the capabilities
+    to support those features.
+  - As an exception, a target identical to an existing tier 1 target except for
+    lower baseline expectations for the OS, CPU, or similar, may propose to
+    qualify as tier 2 (but not higher) without support for `std` if the target
+    will primarily be used in `no_std` applications, to reduce the support
+    burden for the standard library. In this case, evaluation of the proposed
+    target's value will take this limitation into account.
+- The target must build reliably in CI, for all components that Zebra's CI
+  considers mandatory.
+- The approving teams may additionally require that a subset of tests pass in
+  CI, such as enough to build a functional "hello world" program, `./x.py test
+  --no-run`, or equivalent "smoke tests". In particular, this requirement may
+  apply if the tests in question provide substantial value via early detection
+  of critical problems.
+- Building the target in CI must not take substantially longer than the current
+  slowest target in CI, and should not substantially raise the maintenance
+  burden of the CI infrastructure. This requirement is subjective, to be
+  evaluated by the devops team, and will take the community importance
+  of the target into account.
+- Tier 2 targets should, if at all possible, support cross-compiling. Tier 2
+  targets should not require using the target as the host for builds, even if
+  the target supports host tools.
+- Tier 2 targets must not impose burden on the authors of pull requests, or
+  other developers in the community, to ensure that tests pass for the target.
+  In particular, do not post comments (automated or manual) on a PR that derail
+  or suggest a block on the PR based on tests failing for the target. Do not
+  send automated messages or notifications (via any medium, including via `@`)
+  to a PR author or others involved with a PR regarding the PR breaking tests
+  on a tier 2 target, unless they have opted into such messages.
```

Comment:
This isn't what we do for Zebra - we often block PRs on macOS test failures.

Reply:
This seems like a gray area (the text seems valid, but moving to Tier 3 doesn't seems to be an option), let me think it for a while and I'll be back here to discuss it.

Reply:
I just made some suggestions to match our current process.